pkg/ocm/connection.go

```diff
@@ -63,6 +65,16 @@ func (b *ConnectionBuilder) Build() (result *sdk.Connection, err error) {
 		return
 	}
 
+	// overwrite the config URL if the environment variable is set
+	if overrideUrl := os.Getenv("OCM_URL"); overrideUrl != "" {
+		if debug.Enabled() {
+			fmt.Fprintln(os.Stderr, "INFO: OCM_URL is overridden via environment variable. This functionality is considered tech preview and may cause unexpected issues.")
+			fmt.Fprintln(os.Stderr, "      If you experience issues while OCM_URL is set, unset the OCM_URL environment variable and attempt to log in directly to the desired OCM environment.")
+			fmt.Fprintln(os.Stderr, "") // add a newline on purpose to separate from ocm sdk debug output
+		}
+		b.cfg.URL = overrideUrl
```

Comment:
You should check for references to ` config.Save()` like https://github.com/openshift-online/ocm-cli/blob/main/cmd/ocm/get/cmd.go#L126

I believe any command that calls save after `ocm.NewConnection().Build()` will result in that env override being persisted to the config json. Which might be ok in most situations, but is unexpected and best avoided.

Reply:
I'll double check the other commands for this one - but in this case the `cfg` is loaded and then `connection` is built by loading the cfg again. So at least in this case, since we're not getting the `cfg` back from the connection (it's only returning the updated tokens) the URL is not being passed back and is not being updated in the original `cfg` var.

With that said, though, I'll explicitly double check for other instances to make sure we're not inadvertently saving the cfg back from the connection itself and then overwriting the config.

Reply:
Double checked all instances of `config.Save()` that were found and verified that each of them are loading the config separately of the ConnectionBuilder struct, so any changes within ConnectionBuilder would not affect the saving of the refresh/access tokens.

With that said, it probably wouldn't be a terrible idea to further refactor the refresh/access token logic since it's pretty duplicated throughout the codebase, but I would consider that out of scope for this PR specifically.